crates/gitbutler-branch-actions/src/branch.rs

```diff
@@ -481,7 +481,7 @@ pub fn get_branch_listing_details(
     let mut enriched_branches = Vec::new();
     let (diffstats, merge_bases) = {
         let (start, start_rx) = std::sync::mpsc::channel::<(
-            std::sync::mpsc::Receiver<gix::object::tree::diff::ChangeDetached>,
+            std::sync::mpsc::Receiver<gix::diff::tree_with_rewrites::Change>,
```

Comment:
I don't particularly love this and will probably re-introduced the `ChangeDetached` type as alternative name for this.